examples/tutorials/05_ordinary_models.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p>Train <strong>Native</strong> Torch Models?</p><p>word ordinary is quite confusing, I could not understand what it refers to until I read the notebook</p><p> </p><p>Note also that in English titles are written with capitalizing first letters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>